pkg/command/context.go

```diff
@@ -739,6 +773,72 @@ func useCtx(_ *cobra.Command, args []string) error {
 	return nil
 }
 
+var unsetCtxCmd = &cobra.Command{
+	Use:   "unset CONTEXT_NAME",
+	Short: "Unset the active context so that it is not used by default.",
```

Comment:
Can you add the line:
`Args:  cobra.MaximumNArgs(1),`
so there is an error if the user specifies more than one context?